### DISCUSSION TITLE
GenerateResourcesAndImage fails with error - CommandNotFoundException: The term 'mongo' is not recognized as a name of a cmdlet

### DISCUSSION THREAD

**Discussion by plynch66** [actions/virtual-environments#6601] [General questions]
Hi,

I'm trying to generate an Ubuntu2004 image using an Azure VM and I am seeing an error which I haven't previously encountered : 

"azure-arm: Running tests from '/imagegeneration/tests/Databases.Tests.ps1'
    azure-arm: Describing MongoDB
    azure-arm:   [-] mongo 129ms (90ms|39ms)
    azure-arm:    CommandNotFoundException: The term 'mongo' is not recognized as a name of a cmdlet, function, script file, or executable program.
    azure-arm:    Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
    azure-arm:    at <ScriptBlock>, /imagegeneration/tests/Databases.Tests.ps1:7
    azure-arm:   [-] mongod 42ms (40ms|2ms)
    azure-arm:    CommandNotFoundException: The term 'mongod' is not recognized as a name of a cmdlet, function, script file, or executable program.
    azure-arm:    Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
    azure-arm:    at <ScriptBlock>, /imagegeneration/tests/Databases.Tests.ps1:7
    azure-arm: Tests completed in 782ms
    azure-arm: Tests Passed: 0, Failed: 2, Skipped: 0 NotRun: 4"

I have pulled the latest build of the 'runner-images' repo before trying this .

Any clues ?

Paul

**Comment by iancrowl** [comment]
I am also receiving this error and it has been happening for about 2 weeks (11 days)
I have a daily ADO pipeline to build new imaged for my ADO private agents and it has been failing daily.

I would be happy to know if there are any fixes for this.
Thanks

Ian

**Comment by mikhailkoliada** [comment]
Will be fixed by https://github.com/actions/runner-images/pull/6609
